apps/cbportal/base/CBGrid/.macros/page/createmachine/3_createmachine.py

```diff
@@ -47,6 +47,8 @@ def sortName(item):
     popup.addDropdown('Choose Image', 'imageId', dropimages)
     popup.addDropdown('Choose Size', 'sizeId', dropsizes)
```

Comment:
the ui should only support one of the 2 methods, drop the size input and all code related to it